Code/Tools/TestImpactFramework/Runtime/Common/Code/Include/Static/Artifact/Static/TestImpactTargetDescriptor.h

```diff
@@ -45,6 +52,7 @@ namespace TestImpact
     struct TargetDescriptor
     {
         AZStd::string m_name; //!< Build target name.
+        TargetType m_type; //!< The type of target.
```

Comment:
I recommend initializing this enum to a value.
Right the value is indeterminate
```suggestion
        TargetType m_type = TargetType::TestTarget; //!< The type of target.
```